analytical/templatetags/piwik.py

```diff
@@ -98,6 +122,9 @@ def render(self, context):
         if getattr(settings, 'PIWIK_DISABLE_COOKIES', False):
             commands.append(DISABLE_COOKIES_CODE)
 
+        if getattr(settings, 'PIWIK_ASK_FOR_CONSENT', False):
```

Comment:
This still says, `PIWIK` in the constant name. Shouldn't it say `MATOMO`?

Reply:
Actually, this code shouldn't go here at all. As mentioned by my other comments.